src/components/sidebar/components/sidebar-filter-input/sidebar-filter-input.module.css

```
@@ -45,14 +45,14 @@
 }
 
 .filterInput {
+  appearance: none;
+  border-radius: 5px;
+  box-shadow: var(--token-surface-inset-box-shadow);
   composes: g-focus-ring-from-box-shadow from global;
   background-color: transparent;
   border: none;
```

We've been putting `composes` first inside of blocks like this and then placing CSS properties after that, can this be updated to do that here as well??

```suggestion
  composes: g-focus-ring-from-box-shadow from global;
  appearance: none;
  background-color: transparent;
  border: none;
  border-radius: 5px;
  box-shadow: var(--token-surface-inset-box-shadow);
```